certs.jsonnet

```diff
@@ -22,7 +22,6 @@
 
     "automotive-oss.org": [
       "automotive-oss.org",
-      "www.automotive-oss.org",
```

Comment:
is it desired with this commit about OCX?

Reply:
Yes, as www.automotive-oss.org is being served by the same platform as OCX, but it's being (mostly) handled via a cname in DNS  so keeping it here breaks the Certbot workflow, but for 'reasons' we're still doing the redirects for automotive-oss.org at our edge so we still need that cert.

Reply:
:+1